galasa-managers-parent/galasa-managers-workflow-parent/dev.galasa.githubissue.manager/src/main/java/dev/galasa/githubissue/internal/GitHubIssueManagerImpl.java

```diff
@@ -84,21 +89,23 @@ public void initialise(@NotNull IFramework framework, @NotNull List<IManager> al
 		if (galasaTest.isJava()) {
 			Class<?> testClass = galasaTest.getJavaTestClass();
 			
-			this.gitHubIssue = testClass.getAnnotation(GitHubIssue.class); // Set globally as might need later
-			if (this.gitHubIssue == null) { 
-				
-				GitHubIssue gitHubIssue = null;
+			this.classGitHubIssue = testClass.getAnnotation(GitHubIssue.class); // Set globally as might need later
+			if (this.classGitHubIssue == null) { 
+				GitHubIssue methodGitHubIssue = null;
 				for (Method testMethod : testClass.getMethods()) {
-					gitHubIssue = testMethod.getAnnotation(GitHubIssue.class);
-					if (gitHubIssue != null) {
-						required = true;
+					methodGitHubIssue = testMethod.getAnnotation(GitHubIssue.class);
+					if (methodGitHubIssue != null) {
+						methodAnnotated = true;
 						break;
 					}
 				}
 				// Annotation not present on the class or any methods
-				if (!required) {
+				if (!methodAnnotated) {
 					return;
 				}
+			} 
+			if (testClass.getAnnotation(ContinueOnTestFailure.class) != null) {
+				this.continueOnTestFailure = true;
```

Comment:
Hi Will, ContinueOnTestFailure is part of the framework, not a manager, so I'm unsure what I should do here for this and your third comment

Reply:
Hey Jade - yes you are right - the annotation is part of the framework, not a specific manager.  However my point stands.  There is (according to the comments) a CPS property that can be set to override the annotation, so we cannot trust the annotation in isolation, we should provide a method in the framework to allow a manager to obtain the correct state, rather than looking for the annotation directly

Reply:
After some digging I have realised I don't need to find out what the state of ContinueOnTestFailure is as is handled by the framework so I have removed all code trying to access the annotation